index.html

```diff
@@ -7,7 +7,7 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" href="styles.css">
     <script src="challenge.js"></script>
-    <title>Challenge</title>
+    <title>Adam Smith: Porfolio</title>

```

Comment:
Kindly make changes or give the appropriate name for the title header before proceeding to the main HTML mockup. It is important to always give simple header titles to the title tag to make your code nice!

Thank you